sssom/util.py

```diff
@@ -308,7 +317,7 @@ def filter_redundant_rows(df: pd.DataFrame, ignore_predicate=False) -> pd.DataFr
         key = [SUBJECT_ID, OBJECT_ID, PREDICATE_ID]
     dfmax: pd.DataFrame
     dfmax = df.groupby(key, as_index=False)[CONFIDENCE].apply(max).drop_duplicates()
-    max_conf = {}
+    max_conf: Dict[Any, Any] = {}
```

Comment:
better annotation here

Reply:
maybe Dict[str, str] ? But I do not know the general data format of objects expected for this function. Thoughts @matentzn ?

Reply:
No thoughts, sorry.. You will have to check what is expected by the functions its going to.

Reply:
Done!